nss-day-cohort-45/holiday-road-cold-shoulders
# Weather Components Added

## Change:

**Added** 

- weatherCardHTML.js
  - Generates the HTML for each weather card
- weatherList.js
  - Responsible for getting data and placing it on the DOM
- weather.json
  - json DB with test data

**Updated:**

- WeatherProvider.js
  - Added API for the local DB
- main.css
  - Simple styling added for the weather cards
- index.html
  - Added the "card-group" class to the weather-card div for bootstrap styling

## To Test:

Add the following to the main.js and check the cards populated on the DOM:

```
import { weatherList } from "./weather/weatherList.js";
weatherList()
```

In terminal:

`json-server -w weather.json -p 8088`
